src/components/surface/__tests__/CdrSurface.spec.js

```diff
@@ -112,4 +112,40 @@ describe('CdrSurface', () => {
     expect(wrapper.attributes('data-background')).toBe('primary');
     expect(wrapper.element).toMatchSnapshot();
   });
+
+  it('applies background image properties via style attribute', () => {
+    const wrapper = mount(CdrSurface, {
+      props: {
+        backgroundImage: 'path/to/image.jpg',
```

Comment:
https://www.techie-jim.net/2024/05/18/responsive-background-images-with-or-without-image-set-the-proven-way/